dbt/adapters/athena/connections.py

```diff
@@ -64,8 +65,9 @@ class AthenaCredentials(Credentials):
     _ALIASES = {"catalog": "database"}
     num_retries: int = 5
     num_boto3_retries: Optional[int] = None
+    num_iceberg_retries: int = 0
```

Comment:
How about using a value different than 0, for example 3?
What is the reason behind using 0 by default?

Reply:
makes sense, bumped to 3